runtime/bindings/python/src/openvino/ie_api.py

```diff
@@ -3,126 +3,109 @@
-            if arr is not None:
-                arr = np.array(arr)  # Keeping array as numpy array
-                arr_size = int(np.prod(arr.shape))
-                if np.isfortran(arr):
-                    arr = arr.ravel(order="F")
-                else:
-                    arr = arr.ravel(order="C")
-                if arr_size != tensor_desc_size:
-                    raise AttributeError(f"Number of elements in provided numpy array "
-                                         f"{arr_size} and required by TensorDesc "
-                                         f"{tensor_desc_size} are not equal")
-                if arr.dtype != precision_map[precision]:
-                    raise ValueError(f"Data type {arr.dtype} of provided numpy array "
-                                     f"doesn't match to TensorDesc precision {precision}")
-                if not arr.flags["C_CONTIGUOUS"]:
-                    arr = np.ascontiguousarray(arr)
-            elif arr is None:
-                arr = np.empty(0, dtype=precision_map[precision])
-        else:
-            raise AttributeError("TensorDesc can't be None")
-
-        if precision in ["FP32"]:
-            return TBlobFloat32(tensor_desc, arr, arr_size)
-        elif precision in ["FP64"]:
-            return TBlobFloat64(tensor_desc, arr, arr_size)
-        elif precision in ["FP16", "BF16"]:
-            return TBlobInt16(tensor_desc, arr.view(dtype=np.int16), arr_size)
-        elif precision in ["I64"]:
-            return TBlobInt64(tensor_desc, arr, arr_size)
-        elif precision in ["U64"]:
-            return TBlobUint64(tensor_desc, arr, arr_size)
-        elif precision in ["I32"]:
-            return TBlobInt32(tensor_desc, arr, arr_size)
-        elif precision in ["U32"]:
-            return TBlobUint32(tensor_desc, arr, arr_size)
-        elif precision in ["I16"]:
-            return TBlobInt16(tensor_desc, arr, arr_size)
-        elif precision in ["U16"]:
-            return TBlobUint16(tensor_desc, arr, arr_size)
-        elif precision in ["I8", "BIN"]:
-            return TBlobInt8(tensor_desc, arr, arr_size)
-        elif precision in ["U8", "BOOL"]:
-            return TBlobUint8(tensor_desc, arr, arr_size)
-        else:
-            raise AttributeError(f"Unsupported precision {precision} for Blob")
-
-# flake8: noqa: D102
-def blob_from_file(path_to_bin_file: str) -> BlobWrapper:
-    array = np.fromfile(path_to_bin_file, dtype=np.uint8)
-    tensor_desc = TensorDesc("U8", array.shape, "C")
-    return BlobWrapper(tensor_desc, array)
-
-# flake8: noqa: D102
 def tensor_from_file(path: str) -> Tensor:
-    """The data will be read with dtype of unit8"""
+    """Create Tensor from file. Data will be read with dtype of unit8."""
     return Tensor(np.fromfile(path, dtype=np.uint8))
+
+
+def normalize_inputs(py_dict: dict, py_types: dict) -> dict:
+    """Normalize a dictionary of inputs to Tensors."""
+    for k, v in py_dict.items():
```

Comment:
may be rename ```k``` and ```v``` them to ```key``` and ```tensor``` to make it more verbose?

Reply:
Rename to `val`